agent/php_user_instrument_llist.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if LOOKUP_METHOD == LOOKUP_USE_LINKED_LIST
+
+/*
+ * Purpose : Store zend_function's metadata (filename, lineno) in wraprec
+ *           for use in looking up the instrumentation.
+ * 
+ * Params  : 1. The wraprec to update with metadata
+ *           2. The zend function with meta data
+ *
+ * Returns : void
+ */
+static void wraprec_metadata_set(nruserfn_t* wraprec, zend_function* func) {
```

Comment:
This code is a verbatim copy of https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_user_instrument.c#L134-L151.